sdk/textanalytics/ai-text-analytics/src/textAnalyticsResult.ts

```diff
@@ -57,6 +58,13 @@ export interface TextAnalyticsSuccessResult {
    */
   readonly statistics?: TextDocumentStatistics;
 
+  /**
+   * An optional array of warning data corresponding to this document.
```

Comment:
nit: the array isn't optional if it's always present ;)

Reply:
Thanks for the catch, I had originally made this optional, but then decided that since the service is always returning [] I should follow that behavior.